gulpfile.js

```diff
@@ -31,6 +33,7 @@ var WATCH_INTERVAL    = 1000; // in milliseconds
 var ROOT_DIR   = ".";
 var SOURCE_DIR = path.join(ROOT_DIR, "www");
 var DEV_DIR    = path.join(ROOT_DIR, "build-dev");
+var LINT_DIR    = path.join(DEV_DIR, "blog");
```

Comment:
But this excludes the rest of the site too, which should be linted. Is there a way to tell `htmlverify` to exclude/ignore a path?